eng/pipelines/templates/steps/test.yml

```diff
@@ -21,18 +21,26 @@ steps:
   - script: |
       node eng/tools/rush-runner.js build "${{parameters.ServiceDirectory}}" -packages "$(ArtifactPackageNames)" --verbose -p max
     displayName: "Build libraries"
+    env:
+      ${{ if and(eq(variables['System.TeamProject'], 'internal'), or(eq(variables['Build.Reason'], 'IndividualCI'), eq(variables['Build.Reason'], 'BatchedCI'))) }}:
+        RUSH_BUILD_CACHE_CREDENTIAL: $(js-rush-blob-build-cache-credential)
```

Comment:
Similar to what @benbp asked is there a reason we cannot use the built-in DevOps caching task here? https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/cache-v2?view=azure-pipelines

Reply:
> Similar to what @benbp asked is there a reason we cannot use the built-in DevOps caching task here? https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/cache-v2?view=azure-pipelines

The cloud rush build cache also enable local development scenarios: when I build locally I can benefit from cached build output.  I am not sure whether DevOps cache can help this.

Reply:
DevOps cache will definitely not help on local runs, only pipelines.